src/microbe_stage/MicrobeStage.cs

```diff
@@ -514,6 +514,21 @@ public override void OnReturnFromEditor()
         {
             tutorialGUI.EventReceiver?.OnTutorialDisabled();
         }
+        else
+        {
+            // Show day/night cycle tutorial when entering a patch with sunlight
+            if (GameWorld.WorldSettings.DayNightCycleEnabled)
+            {
+                var sunlight = SimulationParameters.Instance.GetCompound("sunlight");
+                var patchSunlight = GameWorld.Map.CurrentPatch!.Biome
```

Comment:
You should call GetCompoundAmount on the Patch type rather than boring into its properties here.